gcloud-java-storage/src/main/java/com/google/cloud/storage/testing/FakeStorageRpc.java

```diff
@@ -37,9 +37,24 @@
 
 /**
  * A bare-bones in-memory implementation of Storage, meant for testing.
- * See LocalGcsHelper.
  *
- * This class is NOT thread-safe.
+ * <p>This class is <i>not</i> thread-safe. It's also (currently) limited in the following ways:
+ * <ul>
+ * <li>Partially Supported
```

Comment:
Sorry, it looks like we merged too fast. I'm making a new PR where we can
discuss additional detail we want in the description.
It's at https://github.com/GoogleCloudPlatform/gcloud-java/pull/1070

On Tue, Jun 21, 2016 at 1:14 PM, Jean-Philippe Martin <jpmartin@google.com>
wrote:

> It means I implemented one of the two methods.
>
> On Tue, Jun 21, 2016 at 1:11 PM, Marco Ziccardi <notifications@github.com>
> wrote:
>
>> In
>> gcloud-java-storage/src/main/java/com/google/cloud/storage/testing/FakeStorageRpc.java
>> <https://github.com/GoogleCloudPlatform/gcloud-java/pull/1069#discussion_r67942420>
>> :
>>
>> >   *
>> > - * This class is NOT thread-safe.
>> > + * <p>This class is <i>not</i> thread-safe. It's also (currently) limited in the following ways:
>> > + * <ul>
>> > + * <li>Partially Supported
>>
>> What does partially supported mean? As a user I would like some more
>> detail on that.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/GoogleCloudPlatform/gcloud-java/pull/1069/files/016e479577e2787fcbfb65e81777403bf994e3ad#r67942420>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AJ-XRV7SErvQWuv3jrQOcVUvVrdGI83Tks5qOEV2gaJpZM4I7Eed>
>> .
>>
>
>